### PR TITLE
add fix for nbp in NorCPM1

### LINKED ISSUE #1114: Dataset problem: Fix nbp in NorCPM1
**Describe the dataset issue**

- Project: CMIP6
- Full dataset description (fill out as many as you know, please):
  - dataset: NorCPM1
  - experiment: historical
  - mip: Lmon
  - grid:gn
  - variable used: nbp
 
- Problems encountered (please describe what the actual problems are: e.g. wrong standard name, issue with dimensions):
multiplying a factor to the data is needed
- Pointer to existing copy of data on ESGF node (it would be very useful if you could provide a physical
fullpath to the file(s) that are causing the problem, e.g. on CEDA Jasmin or DKRZ):
DKRZ: /work/ik1017/CMIP6/data/CMIP6/CMIP/NCC/NorCPM1/historical/r1i1p1f1/Lmon/nbp/gn/v20200724/nbp_Lmon_NorCPM1_historical_r1i1p1f1_gn_185002-201412.nc

### PR DESCRIPTION
**Pull request by LisaBock**
## Description

Closes #1114

## [Checklist](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/contributing.html#checklist-for-pull-requests)

It is the responsibility of the author to make sure the pull request is ready to review. The icons indicate whether the item will be subject to the [🛠 Technical][1] or [🧪 Scientific][2] review.

[1]: https://docs.esmvaltool.org/en/latest/community/review.html#technical-review
[2]: https://docs.esmvaltool.org/en/latest/community/review.html#scientific-review

- [ ] [🧪][2] The new functionality is [relevant and scientifically sound](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/contributing.html#scientific-relevance)
- [ ] [🛠][1] This pull request has a [descriptive title and labels](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/contributing.html#pull-request-title-and-label)
- [ ] [🛠][1] Code is written according to the [code quality guidelines](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/contributing.html#code-quality)
- [ ] [🧪][2] and [🛠][1] [Documentation](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/contributing.html#documentation) is available
- [ ] [🛠][1] [Unit tests](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/contributing.html#tests) have been added
- [ ] [🛠][1] Changes are [backward compatible](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/contributing.html#backward-compatibility)
- [ ] [🛠][1] Any changed [dependencies have been added or removed](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/contributing.html#dependencies) correctly
- [ ] [🛠][1] The [list of authors](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/contributing.html#list-of-authors) is up to date
- [ ] [🛠][1] All [checks below this pull request](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/contributing.html#pull-request-checks) were successful

***

To help with the number pull requests:

- 🙏 We kindly ask you to [review](https://docs.esmvaltool.org/en/latest/community/review.html#review-of-pull-requests) two other [open pull requests](https://github.com/ESMValGroup/ESMValCore/pulls) in this repository

### CODE REVIEW

File: esmvalcore/cmor/_fixes/cmip6/norcpm1.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        Fix data.
+        Parameters
+        ----------
+        cube: iris.cube.Cube
+        Returns
+        -------
+        iris.cube.Cube
+        """
+        metadata = cube.metadata
+        cube *= 29./44.
```
Comment (zklaus) [inline esmvalcore/cmor/_fixes/cmip6/norcpm1.py:19]: Please add a comment explaining the fraction.

Comment (LisaBock) [reply esmvalcore/cmor/_fixes/cmip6/norcpm1.py:19]: @bettina-gier Could you please add this comment? I think you know why to multiply this fraction...

**Comment by remi-kazeroni** [comment]
Hi @bettina-gier, could you please help us document the correction factor for this fix? After that, this PR can be made ready for review.

**Comment by bettina-gier** [comment]
Sorry for the late reply, I'm looking back into it. I think I originally introduced the factor because of this errata from the model family https://errata.es-doc.org/static/view.html?uid=2b26dd51-be64-8d15-b8ca-5a08f7f6b021 so unit conversion stuff. But just running a timeseries of this data I currently get weird results (not depending on this factor) so I need to look at it in more detail.

**Comment by bettina-gier** [comment]
test
This is what I get using NorCPM1 with this fix and comparing to other models (w/ 10-year smoothing). Looks like more things went wrong with NorCPM1 here, especially in the intial period.

**Comment by zklaus** [comment]
Nice to have the erratum. I suggest to add that to the comment as well. That way, we'll see if it gets fixed at some point.

**Comment by remi-kazeroni** [comment]
Note that this [erratum](https://errata.es-doc.org/static/view.html?uid=2b26dd51-be64-8d15-b8ca-5a08f7f6b021) applies to the `NorESM2-LM` and `NorESM2-MM` models but not to the `NorCPM1` model of this PR. I guess we could close this and perhaps open another PR to fix the missing factor for `NorESM2-LM` and `NorESM2-MM` models. What do you think @bettina-gier, @zklaus?

**Comment by bettina-gier** [comment]
I'd close it because from my tests it seems there are problems with the data beyond which could not be fixed by that factor. I personally don't use the AERmon data and the Amon data for co2 is correct, so I don't know if we need this fix until people use the faulty data, just in case it gets fixed for a later version.

**Pull request closed without merging**